libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/session/MapboxTripSession.kt

```diff
@@ -247,31 +253,31 @@ class MapboxTripSession(
     }
 
     private fun updateRawLocation(rawLocation: Location) {
+        locationObservers.forEach { it.onRawLocationChanged(rawLocation) }
```

Comment:
can we move this code to a place where we reproduce the issue with a `Test`.

And then verify that the test passes with the fix.

Reply:
I'm not sure I've got what you mean. What place do you mean?